test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -484,12 +492,15 @@ public void createStudyAndUser() throws SynapseException {
 
         // verify
         verify(participantService, times(2)).createParticipant(any(), any(), any(), anyBoolean());
-        verify(participantService).createParticipant(eq(study), eq(mockUser1.getRoles()), eq(mockUser1), eq(true));
-        verify(participantService).createParticipant(eq(study), eq(mockUser2.getRoles()), eq(mockUser2), eq(true));
+        verify(participantService).createParticipant(eq(study), eq(mockUser1.getRoles()), eq(mockUser1), eq(false));
+        verify(participantService).createParticipant(eq(study), eq(mockUser2.getRoles()), eq(mockUser2), eq(false));
         verify(participantService, times(2)).requestResetPassword(eq(study), eq(mockIdentifierHolder.getIdentifier()));
         verify(mockSynapseClient, times(2)).newAccountEmailValidation(any(), eq(SYNAPSE_REGISTER_END_POINT));
         verify(service).createStudy(study);
         verify(service).createSynapseProjectTeam(TEST_ADMIN_IDS, study);
+        
+        assertTrue(projectCaptor.getValue().getName().startsWith(TEST_PROJECT_NAME));
+        assertTrue(teamCaptor.getValue().getName().startsWith(TEST_TEAM_NAME));
```

Comment:
Don't we also append a random string to the project name and team name? Should we be testing that as well? Otherwise, if we somehow fail to append the random string, this test will still pass, because TEST_PROJECT_NAME starts with itself.